modules/core/src/main/scala/org/tessellation/domain/snapshot/programs/Download.scala

```diff
@@ -34,6 +34,7 @@ import org.tessellation.security.Hashed
 import org.tessellation.security.hash.Hash
 import org.tessellation.security.signature.Signed
 
+import eu.timepit.refined.auto._
```

Comment:
is it needed? there are no other changes in the file 🤔

Reply:
It is not needed and has been removed.